src/pynwb/ndx_photometry/photometry.py

```diff
@@ -5,6 +5,7 @@
 FluorophoresTable = get_class('FluorophoresTable', 'ndx-photometry')
 PhotodetectorsTable = get_class('PhotodetectorsTable', 'ndx-photometry')
 ExcitationSourcesTable = get_class('ExcitationSourcesTable', 'ndx-photometry')
+FiberPhotometryResponseSeries = get_class('FiberPhotometryResponseSeries', 'ndx-photometry')
```

Comment:
Why does this need to be in the photometry.py file?

Reply:
oh, this was a test to see if I could resolve the current import errors... but that didn't help